packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -2,62 +2,99 @@ import Component from '@glimmer/component';
 import { fn } from '@ember/helper';
 import { on } from '@ember/modifier';
 import { action } from '@ember/object';
-import { RenderedLinksToCard } from './stack-item';
+import { RenderedCardForOverlayActions } from './stack-item';
 import { velcro } from 'ember-velcro';
 import { Actions } from '@cardstack/runtime-common';
 import { IconButton } from '@cardstack/boxel-ui';
 import cn from '@cardstack/boxel-ui/helpers/cn';
 import { type TrackedArray } from 'tracked-built-ins';
 import type { MiddlewareState } from '@floating-ui/dom';
 import type { Card } from 'https://cardstack.com/base/card-api';
+import { cardTypeDisplayName } from '@cardstack/runtime-common';
+import { and, eq, not } from '@cardstack/host/helpers/truth-helpers';
+import { bool } from '@cardstack/boxel-ui/helpers/truth-helpers';
```

Comment:
Looks like we can benefit from using the truth-helpers in boxel-ui '@cardstack/boxel-ui/helpers/truth-helpers';

`and` handles a list of arguments rather than just 2. `eq` and `not` seems roughly identical

Reply:
That's good to know, thanks. It looks like we can remove the `packages/host/app/helpers/truth-helpers.ts` file and use the other one instead